gotrue/_async/gotrue_client.py

```diff
@@ -483,10 +484,15 @@ async def sign_out(self) -> None:
         There is no way to revoke a user's access token jwt until it expires.
         It is recommended to set a shorter expiry on the jwt for this reason.
         """
-        session = await self.get_session()
-        access_token = session.access_token if session else None
-        if access_token:
-            await self.admin.sign_out(access_token)
+        try:
+            session = await self.get_session()
+            access_token = session.access_token if session else None
+            if access_token:
+                await self.admin.sign_out(access_token)
+        except AuthApiError:
+            pass
+        except Exception:
+            raise
```

Comment:
This shouldn't be needed - if the error is anything except an AuthApiError it should be raised anyways